src/components/Chip/Chip.scss

```diff
@@ -0,0 +1,34 @@
+@import "~vanilla-framework/scss/settings_spacing";
+
+.p-chip {
+  // XXX Ant - 22.06.20: hardcoding @extend %small-text; due to:
+  // https://github.com/canonical-web-and-design/vanilla-framework/issues/3133
+  font-size: .875rem;
+  line-height: 1rem;
+
+  background-color: rgba(0, 0, 0, 0.05);
+  border-radius: 1rem;
+  cursor: pointer;
+  display: inline-block;
+  margin-bottom: $sp-unit + $spv-inner--small - 2 * map-get($nudges, nudge--small);
```

Comment:
🤯

Reply:
I know, I hate it too but copied from Vanilla.

Reply:
Removed as 0 is much more atomic.